markdown/assignments/pool_b/nahkampf/index.md

```diff
@@ -3,14 +3,24 @@ type: assignment
 title: "Nahkampf"
 author: "Andre Matutat (FH Bielefeld)"
 points: 2
-weight: 1
+weight: 2
 hidden: true
 ---
 
+## Ziel
+
+In dieser Aufgabe implementieren Sie ein einfaches Nahkampfsystem, damit der Held gegen die Monster kämpfen kann.
+
+## Voraussetzung
+
+Um diese Aufgae lösen zu können, müssen Sie vorher `["Monster"]({{< ref "/assignments/pool_b/monster" >}})`{=markdown} implementiert haben.
 
 ## Nahkampf
 
--   Implementieren Sie eine rudimentäre Form eines Kampf-Systems. Dabei soll es zum Kampf kommen, wenn der Held und ein Monster auf demselben Feld stehen.
--   Bei einem erfolgreichen Angriff wird dem angegriffenen Gegner ein bestimmter Schaden zugefügt (dieser verliert dann Lebenspunkte).
--   Ob ein Angriff erfolgreich ist, kann mit einem Zufallsgenerator bestimmt werden: Beispielsweise könnten 60% aller Angriffe erfolgreich sein.
--   Wird der Held von einem Monster getroffen, soll er ein eine kurze Distanz zurückgeschleudert werden. Beachten Sie dabei, dass der Held nicht durch eine Wand bewegt wird.
+Implementieren Sie eine rudimentäre Form eines Kampf-Systems. Dabei soll es zum Kampf kommen, wenn der Held und ein Monster auf demselben Feld stehen (sich berühren).
```

Comment:
technisch wäre "berühren" aber auch, wenn sie in direkt aneinandergrenzenden feldern stehen, oder? sollten wir den teil in klammern streichen?

Reply:
ja, hast recht